python/paddle/fluid/contrib/quantize/quantize_transpiler.py

```diff
@@ -84,7 +84,8 @@ def __init__(self,
                  activation_bits=8,
```

Comment:
Because Transpilers are deprecated. I think these changes about QuantizeTranspiler in this PR should update on `QuantizationTransformPass` and `QuantizationFreezePass` first.  After that, you can choose to make the same update on `QuantizeTranspiler`.

Reply:
Please update the Pass, thanks @achao2013